scripts/globals/job_utils/summoner.lua

```diff
@@ -111,10 +118,12 @@ local function getBaseMPCost(player, ability)
 
     local baseMPCost = nil
 
-    if ability:getAddType() == xi.addType.ADDTYPE_ASTRAL_FLOW then
-        baseMPCost = player:getMainLvl() * 2
-    elseif ability ~= nil then
-        baseMPCost = baseMPCostMap[ability:getID()]
+    if ability then
```

Comment:
When/how is ability `nil` ? hmm...

Reply:
Hopefully never, but I just cleaned up the existing logic